### ISSUE TITLE
Running the database seeding program kills the Ceramic node

### ISSUE THREAD

**Issue by dysbulic** [MetaFam/TheGame#998]
### What happened?

Running `yarn docker:clean && docker-compose up --build && yarn hasura:seed-local-db` inserts a couple hundred users into the database. When a user is inserted it triggers pulling their profile from Ceramic to update the cache.

Requests are rate-limited to ten concurrently by [`bottleneck`](https://github.com/SGrondin/bottleneck#readme).

About 50% of the way through the profile updates you start seeing:

```
Error: Update Error: HTTP request to 'http://ceramic.metagame.wtf:7007/api/v0/streams' failed with status 'Internal Server Error': {"error":"request to http://ipfs:5011/api/v0/block/stat?timeout=30000ms&arg=bafyreidfieno6ba5pk4x6efv5cpazqcdyee7vvvlk3wgkzap3bp5mwqu6q failed, reason: connect ECONNREFUSED 10.74.3.171:5011"}
```

Eventually the lack of an IPFS endpoint catches up with Ceramic daemon & it starts returning:

```
Update Error: HTTP request to 'http://ceramic.metagame.wtf:7007/api/v0/streams&#39; failed with status 'Internal Server Error';
```

### What did you expect to happen?

The daemon to not fall over. 😅

### How can we reproduce the problem (as minimally as possible)?

`yarn docker:clean && docker-compose up --build && yarn hasura:seed-local-db` using the Hasura configuration and seeding script from [`feat/glaze-lookup`](https://github.com/MetaFam/TheGame/tree/feat/glaze-lookup).

### Is there anything else we need to know?

There is a known memory leak in the JavaScript IPFS daemon that is likely the root cause of this failure. Ceramic relies on DAG-JOSE which is currently only available for the JavaScript implementation.

Support for the more scalable Go version is underway and fast approaching release. Word from their Discord is they expect the integration to be released shortly after the holidays.

Assuming they release [Docker images](https://github.com/ceramicnetwork/go-ipfs-daemon) for the node, upgrading our installation should be relatively straightforward.

**Comment by polimyl** [comment]
the nodes are currently getting quite little resources because of cost concerns. this should have been enough for our current usage but bombarding it for a while wasnt considerer in this. could be easily solved by just giving it more ressources

**Comment by dysbulic** [comment]
I'm going to ratchet down the rate limiting and see what effect that has.

I've not dug into the logs yet to get a sense of why it's dying. I was just going off the fact that whenever someone complains of their server dying on the Ceramic Discord, they talk about that leak.

**Comment by dysbulic** [comment]
> the nodes are currently getting quite little resources because of cost concerns.

¿Are we able to configure it to ramp up when the load increases and then ramp back down?

¿Is it operating under significant limits currently?

**Comment by dysbulic** [comment]
It turns out that querying Ceramic for profile information for a newly created user wasn't rate limited & that's the main reason the seeding script was killing the daemon.

That is fixed in #943.
